src/app/users/[id]/components/ApplicationDetailModal.tsx

```diff
@@ -0,0 +1,226 @@
+            status === '상태' && 'text-gray-500',
+            status === '취소' && 'text-black bg-gray-300'
+          )}
+        >
+          {status}
+          {status != '취소' && (
+            <div className="px-2 text-black">
+              <IcoArrow direction="down" className="ml-2" />
+            </div>
+          )}
+        </label>
+
+        <div className="z-10 absolute left-0 mt-2 w-full bg-white border border-gray-200 rounded shadow-md opacity-0 peer-checked:opacity-100 peer-checked:visible invisible transition-opacity">
+          <button
+            onClick={() => setStatus('승인')}
+            className="block px-4 py-2 text-green hover:bg-gray-100"
+          >
+            승인
+          </button>
+          <button
+            onClick={() => setStatus('반려')}
+            className="block px-4 py-2 text-red hover:bg-gray-100"
+          >
+            반려
+          </button>
+        </div>
+      </div>
+    );
+  };
+
+  return (
+    <section className="w-100 h-100 flex flex-col fixed inset-0 flex items-center justify-center bg-black z-12 bg-opacity-30">
+      <div className="w-500 h-500 bg-white p-5 rounded-lg shadow-lg">
+        {isLoading && <LoadingSpinner />}
+        <section className="p-4 font-bold text-lg">프로그램 신청 상세</section>
+
+        <section className="flex my-3 px-3">
+          <div className="pr-10 pl-2">
+            <img src="/img_404.png" alt={`${'1'}`} width={100} height={100} />
+          </div>
+          <div>
+            <div className="flex my-2">
+              <h2 className="font-bold text-lg">{data?.programInfo.title}</h2>
+            </div>
+
+            <div className="text-gray-500">
+              {formatDateTime(data?.programInfo.createdAt)}
+            </div>
+          </div>
+        </section>
+
+        <section className="px-3 py-3">
+          <div className="py-3 font-bold text-lf">신청자 정보</div>
+          <table>
+            <tr>
+              <td className="text-bold text-gray-500 py-2">이름</td>
+              <td className="px-4">{data?.applierInfo.name}</td>
+            </tr>
+            <tr>
+              <td className="text-bold text-gray-500 py-2">핸드폰 번호</td>
+              <td className="px-4">
+                {formatHyphenPhone(data?.applierInfo.phone)}
+              </td>
+            </tr>
+            <tr>
+              <td className="text-bold text-gray-500 py-2">성별</td>
+              <td className="px-4">{data?.applierInfo.gender}</td>
+            </tr>
+            <tr>
+              <td className="text-bold text-gray-500 py-2">생년월일</td>
+              <td className="px-4">{data?.applierInfo.birthday}</td>
+            </tr>
+            <tr>
+              <td className="text-bold text-gray-500 py-2">주소</td>
+              <td className="px-4">
+                {data?.applierInfo.address} {data?.applierInfo.addressDetail}
+              </td>
+            </tr>
+          </table>
+        </section>
+
+        {hasAdditionalInfo() && (
```

Comment:
간단한 조건이어서 굳이 함수로 빼지 않아도 될 것 같습니다.
```suggestion
        {data?.answers.length > 0 || data?.attachmentFileIds.length > 0 && (
```